content/recipes/prisma.md

```diff
@@ -537,4 +530,9 @@ This controller implements the following routes:
 
 In this recipe, you learned how to use Prisma along with NestJS to implement a REST API. The controller that implements the routes of the API is calling a `PrismaService` which in turn uses Prisma Client to send queries to a database to fulfill the data needs of incoming requests.
 
-If you want to learn more about Prisma, be sure to check out the [documentation](https://www.prisma.io/docs/).
+If you want to learn more about using NestJS with Prisma, be sure to check out the following resources:
+
+- [NestJS & Prisma](https://wwww.prisma.io/nestjs)
+- [Ready-to-run example projects for REST & GraphQL]()
```

Comment:
An URI is missing here

Reply:
Hey Johannes, nice to see you around again 😄 thanks for catching this, just added the URL pointing to the `prisma-examples` repo 👍

Reply:
Thanks for the welcoming words ;)